docs/ops/pooling/MaxPool_1.md

```diff
@@ -67,20 +67,192 @@
 
```

Comment:
Please correct `Note`. *pads_begin* and *pads_end* attributes are ignored when *auto_pad* is not equal to  *explicit*.